se/commands/compare_versions.py

```diff
@@ -67,7 +66,7 @@ def compare_versions(plain_output: bool) -> int:
 				# If we work directly on the real repo, ctrl + c may leave it in a stashed state unexpectedly.
```

Comment:
oops, just saw this comment :/.

Reply:
Why don't we use `os.makedirs(…, exist_ok=True)` before running this command to get around the limitation pointed out in the comment.